__tests__/app.test.js

```diff
@@ -233,3 +233,21 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe('GET /api/users', () => {
+  test('200: responds with an array of all user objects', () => {
+    return request(app)
+    .get('/api/users')
+    .expect(200)
+    .then((response) => {
+      response.body.users.forEach((user) => {
```

Comment:
if this didn't come back with any data (empty array) this would give you a false positive